tally_ho/apps/tally/models/reconciliation_form.py

```diff
@@ -77,5 +77,29 @@ def number_ballots_expected(self):
                 self.number_unstamped_ballots -
                 self.number_invalid_votes)
 
+    @property
+    def number_ballots_inside(self):
```

Comment:
the word "inside" is ambiguous to me, what does that mean? Is it received in any ballot box?

Reply:
I will update this, should read `number_ballots_inside_the_box` instead

Reply:
Done